tools/android_lint/project.xml

```diff
@@ -0,0 +1,81 @@
+<!-- THIS FILE IS GENERATED. PLEASE USE THE INCLUDED DART PROGRAM  WHICH -->
+<!-- WILL AUTOMATICALLY FIND ALL .java FILES AND INCLUDE THEM HERE       -->
```

Comment:
Should you find all Java files, or only the Java files listed in BUILD.gn?

Reply:
We could use BUILD.gn - in which case we could probably write this as a build rule.  I don't really have a strong opinion on that - would we want unlinted files in the repo that aren't part of the build though?

Reply:
I guess the question is, why would a Java file exist without being a part of BUILD.gn....and if no such situation can occur, then why not just yank the Java file paths out of BUILD.gn?